src/rpc/util.h

```diff
@@ -324,8 +324,11 @@ struct RPCResult {
     std::string ToStringObj() const;
     /** Return the description string, including the result type. */
     std::string ToDescriptionString() const;
-    /** Check whether the result JSON type matches. */
-    bool MatchesType(const UniValue& result) const;
+    /** Check whether the result JSON type matches.
+     * Returns value where .isTrue() passes is type matches,
```

Comment:
```suggestion
     * Returns true if type matches,
```

nit: Use simple English?